mkat_fpga_tests/test_cbf.py

```diff
@@ -607,8 +609,9 @@ def test_sensor_values(self):
         rct.start()
         rct.until_synced()
 
-        ## 1. Request a list of available sensors using KATCP command
-        ## 2. Confirm the CBF replies with a number of sensor-list inform messages
+        # 1. Request a list of available sensors using KATCP command
+        # 2. Confirm the CBF replies with a number of sensor-list
+        # inform messages
```

Comment:
Perhaps indent 'inform messages' to be inline with the start of the list? i.e.

    # 2. Confirm blab blah
    #    inform messages

Should make it easier to read.